README.md

```diff
@@ -1,16 +1,53 @@
-# Prometheus Stack
-Install the [kube-prometheus-stack](https://github.com/prometheus-operator/kube-prometheus), de-facto standard for monitoring. 
+# About this module
 
+This module can install Prometheus Grafana Thanos.
+
+## How to change infrastructure
+
+ * New namespace will be create (by default name "monitoring")
+ * New 3 application for ArgoCD (grafana, prometheus, thanos) 
+ * New ingress and dns records will be create (thanos.domain.name, grafana.domain.name). You can disable create ingress if provide additional config (grafana_conf = {ingress.enabled = false} and thanos_conf = {queryFrontend.ingress.enabled = false}
+ * By default, thanos backend will be create s3 bucket "<domain_name>-thanos and IAM policy. 
+## Prometheus
+Install the [kube-prometheus](https://github.com/bitnami/charts/tree/master/bitnami/kube-prometheus), de-facto standard for monitoring.
+## Grafana
+Install the [grafana](https://github.com/grafana/helm-charts/tree/main/charts/grafana), web dashboarding system
+## Thanos
+Install the [thanos](https://github.com/bitnami/charts/tree/master/bitnami/thanos), long term storage capabilities for prometheus
 ## Example
 ``` hcl
 module "prometheus" {
-  source        = "git::https://github.com/provectus/sak-prometheus.git"
-  cluster_name      = module.kubernetes.cluster_name
-  argocd            = module.argocd.state
-  domains           = local.domain
+  depends_on      = [module.argocd]
```

Comment:
is it require? you already use outputs of argocd module as inputs for this module